astroid/rebuilder.py

```diff
@@ -104,13 +104,17 @@ def __init__(
             self._parser_module = parser_module
         self._module = self._parser_module.module
 
-    def _get_doc(self, node: T_Doc) -> Tuple[T_Doc, Optional[str]]:
+    def _get_doc(self, node: T_Doc) -> Tuple[Optional[str], Optional[nodes.Const]]:
+        """Returns the docstring of a node both in str and nodes.Const.
+
+        The constant will only be returned in python 3.8+ because the improved ast parsers
+        gives as additional information about line numbers.
+        """
         try:
             if PY37_PLUS and hasattr(node, "docstring"):
                 doc = node.docstring  # type: ignore[union-attr,attr-defined] # mypy doesn't recognize hasattr
-                return node, doc
+                return doc, None
```

Comment:
When is this called? AFAIK no AST nodes have a `docstring` attribute.
It was added here https://github.com/PyCQA/astroid/commit/2c655de34603e5a0f88b369d3e4b68556fdb0641 together with `typed_ast` but even that doesn't seem to use `docstring`.

Am I missing something?

Reply:
I just checked the coverage report. So far these lines are not covered.

Reply:
We might be good with removing this? Should probably be done in a separate PR though.

Reply:
Opened #1289